experimental/uefi/app/src/main.rs

```diff
@@ -85,6 +89,7 @@ fn main(handle: Handle, system_table: &mut SystemTable<Boot>) -> Status {
 //  Normally does not return, unless an error is raised.
 fn serial_echo(handle: Handle, bt: &BootServices, index: usize) -> Result<!, uefi::Error<()>> {
```

Comment:
Looks like this will conflict with #2745. It is probably a good idea to wait for that to be merged and then rebase on top of it.

Reply:
sgtm